test/integration/proposal-genericSchemeEnsRegistry.e2e.ts

```diff
@@ -16,6 +16,7 @@ describe("Proposals ENS Registry", () => {
 
   it("Create a Generic Scheme ENS Registry proposal and check that the data is submitted correctly", async () => {
     await gotoDaoSchemes(daoAddress);
+    await hideCookieAcceptWindow();
```

Comment:
why is this needed ?

Reply:
For some reason this test failed on Travis and locally because the cookies window blocked it. Anyway it's harmless it just hides the cookies prompt.

Reply:
@roienatan it can be a problem for other tests as well? Can you show the video, how does it looks like?

Reply:
As I saw it's not consistent. Will explain you when we talk.